File: src/main/java/chat/sprchat/state/LoadedMessage.java
package chat.sprchat.state;

import chat.sprchat.state.orm.Message;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.sql.Time;
import java.sql.Timestamp;
import java.util.Date;

@Getter @Setter @AllArgsConstructor
public class LoadedMessage
{
    String user;
    String message;
    Date date;
}

File: src/main/java/chat/sprchat/state/LoadedMessage.kt
package chat.sprchat.state

import java.util.*

data class LoadedMessage(
        private var user: String,
        private var message: String,
        private var date: Date,
        private val id: Long
){
    fun getUser(): String{
        return user
    }
    fun setUser(_user: String){
        user = _user
    }
    fun getDate(): Date{
        return date
    }
    fun setDate(_date: Date){
        date = _date
    }
    fun getMessage(): String{
        return message
    }
    fun setMessage(_message: String){
        message = _message
    }
}